scaleapi/api.py

```diff
@@ -109,9 +109,28 @@ def _api_request(
         json = None
         if res.status_code == 200:
             json = res.json()
+        elif res.status_code == 409:
+            retry_history = res.raw.retries.history
+            # Example RequestHistory tuple
+            # RequestHistory(method='POST',
+            #   url='/v1/task/imageannotation',
+            #   error=None,
+            #   status=409,
+            #   redirect_location=None)
+            if retry_history != ():
+                # See if the first retry was a 500 error
+                if retry_history[0][3] == 500:
+                    uuid = body["unique_id"]
+                    newUrl = f"https://api.scale.com/v1/tasks?unique_id={uuid}"
+                    # grab task from api
+                    newRes = self._http_request(
+                        "GET", newUrl, headers=headers, auth=auth
+                    )
+                    json = newRes.json()["docs"][0]
```

Comment:
To confirm, will this `json` be the same as what we return after task creation, right?

Reply:
yep